storage/tianmu/common/common_definitions.cpp

```diff
@@ -34,52 +34,64 @@ void PushWarning(THD *thd, Sql_condition::enum_severity_level level, uint code,
 
 // TODO:Here for args int `type`, we do not use enum directly as it'll caused compiling failed for the dependent
 // package. It will be changed back to enum type later
+// We use std::exception() but not commong::Exceptions here as it will raise infos like "internal error in storage
+// tianmu ..." on mysql client, this is not user friendly. Users has already get error msg from push_warning() func.
 void PushWarningIfOutOfRange(THD *thd, std::string col_name, int64_t v, int type, bool unsigned_flag) {
   // below `0` is for min unsigned value.
   switch (type) {
     case 1: {  // MYSQL_TYPE_TINY
       if (unsigned_flag && (static_cast<uint64_t>(v) > TIANMU_TINYINT_MAX)) {
         PushWarning(thd, Sql_condition::SL_WARNING, ER_WARN_DATA_OUT_OF_RANGE,
                     getErrMsg(col_name, 0, TIANMU_TINYINT_MAX, unsigned_flag, v).c_str());
+        throw std::exception();
```

Comment:
The std::exception is not the best choice, we can throw a specific exception. For example: DataTypeNotSupported or something related.
The some reseaon for that:
1: The exception is the base class, we should not throw that as our first choice; 
2: The eeception mechanism is havery-cost way;
3: The catch statement will be take more statement to catch this.